stable/golang/golang_legacy.jsonnet

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
this file should contain the runtimes for `go1.10`, `go1.11` and `go1.12` :)

Reply:
Lol. Missed that, thanks =).